src/main/java/appeng/container/implementations/ContainerIOPort.java

```diff
@@ -37,7 +37,7 @@
 public class ContainerIOPort extends ContainerUpgradeable
 {
 
-	final TileIOPort ioPort;
+	private final TileIOPort ioPort;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "ioPort" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)